tests/js/environment.js

```diff
@@ -1,6 +1,6 @@
 var fluid = require("infusion");
 
-fluid.defaults("gpii.pouch.tests.environment", {
+fluid.defaults("gpii.test.pouch.environment", {
```

Comment:
I will look at it, but we should discuss in our chat tomorrow so I can make the update Friday morning.